src/plugins/links/server/content_management/links_storage.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { SOContentStorage } from '@kbn/content-management-utils';
+import { CONTENT_ID } from '../../common';
+import type { LinksCrudTypes } from '../../common/content_management';
+import { cmServicesDefinition } from '../../common/content_management/cm_services';
+
+export class LinksStorage extends SOContentStorage<LinksCrudTypes> {
+  constructor() {
+    super({
```

Comment:
heads up! I just merged this improvement to the cm utils https://github.com/elastic/kibana/pull/166919 and unfortunately you'll need to update this or ts will fail. sorry for the inconvenience

Reply:
Fixed in [79eb54d](https://github.com/elastic/kibana/pull/166896/commits/79eb54df3527a511eb0d1e58e73d170811c5822e). Thanks for the heads up!